src/clojure_tictactoe/game/players.clj

```diff
@@ -9,3 +11,11 @@
 
 (defn player-swap []
   (swap! player-atom switch-player))
+
+(defn human-player-move [board]
+  (input-getter/get-player-move board))
+
+(defn choose-player-function [player]
+  (if (= player "x")
+    human-player-move
```

Comment:
How does this work? Shouldn't it be called with parens and passed an argument?

Reply:
It returns the function that needs to be called depending on which player's move it is. Then that function is called with parens and arguments.

Reply:
Ah I see.